src/OceanSeaIceModels/InterfaceComputations/similarity_theory_turbulent_fluxes.jl

```diff
@@ -554,21 +592,37 @@ end
 Base.summary(ss::SplitStabilityFunction) = "SplitStabilityFunction"
 Base.show(io::IO, ss::SplitStabilityFunction) = print(io, "SplitStabilityFunction")
 
+@inline (ψ::SplitStabilityFunction)(ζ) = stability_profile(ψ, ζ)
+
 @inline function stability_profile(ψ::SplitStabilityFunction, ζ)
     Ψ_stable = stability_profile(ψ.stable, ζ)
     Ψ_unstable = stability_profile(ψ.unstable, ζ)
     stable = ζ > 0
     return ifelse(stable, Ψ_stable, Ψ_unstable)
 end
 
-function atmosphere_sea_ice_stability_functions(FT=Oceananigans.defaults.FloatType)
+function atmosphere_sea_ice_stability_functions(FT=Oceananigans.defaults.FloatType;
+                                                tabulate_stability_functions = true,
+                                                tabulation_ζ_range = (-200, 200),
```

Comment:
```suggestion
                                                tabulate_stability_functions = false,
                                                tabulation_ζ_range = (-100, 100),
```
unfortunately this does not work, the error is too large...